frontend/src/lib/components/common/button/model.ts

```diff
@@ -20,12 +20,27 @@ export namespace ButtonType {
 		xl: 'text-xl'
 	} as const
 
-	export const SpacingClasses: Record<ButtonType.Size, string> = {
-		xs: 'px-3 py-1.5',
-		sm: 'px-3 py-1.5',
-		md: 'px-3 py-1.5',
-		lg: 'px-4 py-2',
-		xl: 'px-4 py-2'
+	export const SpacingClasses: Record<ButtonType.Size, Record<ButtonType.Variant, string>> = {
+		xs: {
+			border: 'px-3 py-[6px]',
```

Comment:
What was the issue? 6,7,8,9 px seems like magic numbers

Reply:
They were 6px and 8px (given in rem) but the bordered versions were 2px taller because of the border, so I added 1-1px on the y axis.